docs/root/intro/arch_overview/security/threat_model.rst

```diff
@@ -15,16 +15,33 @@ highest priority concerns. Availability, in particular in areas relating to DoS
 exhaustion, is also a serious security concern for Envoy operators, in particular those utilizing
 Envoy in edge deployments.
 
-The Envoy availability stance around CPU and memory DoS, as well as Query-of-Death (QoD), is still
-evolving. We will continue to iterate and fix well known resource issues in the open, e.g. overload
-manager and watermark improvements. We will activate the security process for disclosures that
-appear to present a risk profile that is significantly greater than the current Envoy availability
-hardening status quo. Examples of disclosures that would elicit this response:
-
-* QoD; where a single query from a client can bring down an Envoy server.
-
-* Highly asymmetric resource exhaustion attacks, where very little traffic can cause resource exhaustion,
-  e.g. that delivered by a single client.
+We will activate the security release process for disclosures that meet the following criteria:
+
+* All issues that lead to loss of data confidentiality or integrity trigger the security release process.
+* An availability issue, such as Query-of-Death (QoD) or resource exhaustion needs to meet all of the
+  following criteria to trigger the security release process:
+  
+  - A component tagged as hardened is affected (see `Core and extensions`_ for the list of hardened components).
+    
+  - The type of traffic (upstream or downstream) that exhibits the issue matches the component's hardening tag.
+    I.e. component tagged as “hardened to untrusted downstream” is affected by downstream request.
+    
+  - A resource exhaustion issue needs to meet these additional criteria:
+    
+    + Not covered by an existing timeout or where applying short timeout values is impractical and either
+      
+      + Per-request memory use 100x or more above the configured header or high watermark limit. I.e. 10 KiB
+        client request leading to 1 MiB bytes of memory consumed by Envoy or
+      
+      + CPU exhaustion where 100 or fewer HTTP/1 connections or HTTP/2 streams with requests sent at 50Mbit/s
```

Comment:
What is the expected throughput per CPU of Envoy?  50Mbit/s seems within a factor of 2 of expected performance, whic seems too tight.

Reply:
Yeah, I would drop this by an order of magnitude. We can always make these stricter over time.

Reply:
done